phoenix-client-parent/phoenix-client/pom.xml

```diff
@@ -85,10 +85,5 @@
       <artifactId>log4j</artifactId>
       <scope>runtime</scope>
     </dependency>
-    <dependency>
```

Comment:
I think that we'd best keep the old client unchanged for backwards compatibility reasons.